apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/DataLayoutStrategyGeneratorSparkApp.java

```diff
@@ -41,15 +46,39 @@ protected void runInner(Operations ops) throws Exception {
         strategies.stream().map(Object::toString).collect(Collectors.joining(", ")));
     StrategiesDao dao = StrategiesDaoTableProps.builder().spark(spark).build();
     dao.save(fqtn, strategies);
+    if (outputFqtn != null && !strategies.isEmpty()) {
+      List<String> rows = new ArrayList<>();
+      for (DataLayoutStrategy strategy : strategies) {
+        rows.add(
+            String.format(
+                "('%s', current_timestamp(), %f, %f, %f, %f, '%s')",
+                fqtn,
+                strategy.getCost(),
+                strategy.getGain(),
+                strategy.getEntropy(),
+                strategy.getScore(),
+                SerDeUtil.toJsonString(strategy)));
```

Comment:
Nit: Instead of JsonString should be consider type map<string, string>?
key: class variable name

Reply:
i think it's easier to work with json string as we add more fields, all the values that will be used in ranking are flattened

Reply:
Do we need this right now? Maybe we can add it later?

Reply:
ok, I can make it lightweight and remove for now